File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test1/Java/Instrmt.java
package object;



// Class for tag Instrmt
public class Instrmt {

  String Sym;
  String ID;
  String IDSrc;


  // default constructor
  public Instrmt ( String Sym, String ID, String IDSrc ) {
    this.Sym = Sym;
    this.ID = ID;
    this.IDSrc = IDSrc;
  
  }


  public Instrmt ( ) {   
  
this.Sym = "IBM";
this.ID = "459200101";
this.IDSrc = "1";
  
  }
      

} 

File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test1/CPlus/Instrmt.cpp
#include <iostream> 

using namespace std;

class Instrmt {

	private:
  	string Sym_;
  	string ID_;
  	string IDSrc_;
 	
 	public: 

  	// default constructor
	Instrmt () {
		Sym_ = "IBM";
		ID_ = "459200101";
		IDSrc_ = "1";
  	
  	}
  
  	// by parameters
  	Instrmt (		string Sym_p, string ID_p, string IDSrc_p){
	  
		Sym_ = Sym_p;
		ID_ = ID_p;
		IDSrc_ = IDSrc_p;
	  
  	}
};